<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.3 (Python 3.8.10 on linux)" generated="20220314 11:38:06.789" rpa="true" schemaversion="2">
<suite id="s1" name="Tasks" source="/home/nhat/Jenkins_Robot/tasks.robot">
<test id="s1-t1" name="Order robots from RobotSpareBin Industries Inc">
<kw name="Open the robot order website">
<kw name="Open Available Browser" library="RPA.Browser.Selenium">
<arg>https://robotsparebinindustries.com/#/robot-order</arg>
<doc>Attempts to open a browser on the user's device from a set of
supported browsers. Automatically downloads a corresponding webdriver
if none is already installed.</doc>
<msg timestamp="20220314 11:38:11.271" level="INFO">Creating an instance of the Chrome WebDriver.</msg>
<msg timestamp="20220314 11:38:11.295" level="INFO">Created download root directory: /home/nhat/.robocorp/webdrivers</msg>
<msg timestamp="20220314 11:38:11.295" level="INFO">Created symlink directory: /home/nhat/.robocorp/webdrivers</msg>
<msg timestamp="20220314 11:38:11.308" level="INFO">Creating an instance of the Chrome WebDriver.</msg>
<msg timestamp="20220314 11:38:13.731" level="INFO">Created Chrome browser with arguments: --disable-dev-shm-usage --disable-web-security --allow-running-insecure-content --no-sandbox</msg>
<msg timestamp="20220314 11:38:13.731" level="INFO" html="true">&lt;p&gt;Attempted combinations:&lt;/p&gt;&lt;div class="doc"&gt;&lt;table&gt;&lt;tr&gt;&lt;th&gt;Browser&lt;/th&gt;&lt;th&gt;Download&lt;/th&gt;&lt;th&gt;Error&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;False&lt;/td&gt;&lt;td&gt;Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Chrome&lt;/td&gt;&lt;td&gt;True&lt;/td&gt;&lt;td&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;</msg>
<msg timestamp="20220314 11:38:13.731" level="INFO">Opening url 'https://robotsparebinindustries.com/#/robot-order'</msg>
<status status="PASS" starttime="20220314 11:38:11.269" endtime="20220314 11:38:15.405"/>
</kw>
<kw name="Maximize Browser Window" library="RPA.Browser.Selenium">
<doc>Maximizes current browser window.</doc>
<kw name="Capture Page Screenshot" library="RPA.Browser.Selenium">
<doc>Takes a screenshot of the current page and embeds it into a log file.</doc>
<status status="FAIL" starttime="20220314 11:38:15.510" endtime="20220314 11:38:15.516"/>
</kw>
<msg timestamp="20220314 11:38:15.518" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: Execution terminated by signal</msg>
<status status="FAIL" starttime="20220314 11:38:15.406" endtime="20220314 11:38:15.519"/>
</kw>
<status status="FAIL" starttime="20220314 11:38:11.266" endtime="20220314 11:38:15.521"/>
</kw>
<kw name="Get orders">
<var>${orders}</var>
<status status="NOT RUN" starttime="20220314 11:38:15.525" endtime="20220314 11:38:15.526"/>
</kw>
<for flavor="IN">
<var>${row}</var>
<value>@{orders}</value>
<iter>
<var name="${row}">${row}</var>
<kw name="Close the annoying modal">
<status status="NOT RUN" starttime="20220314 11:38:15.529" endtime="20220314 11:38:15.531"/>
</kw>
<kw name="Fill the form">
<arg>${row}</arg>
<status status="NOT RUN" starttime="20220314 11:38:15.533" endtime="20220314 11:38:15.534"/>
</kw>
<kw name="Preview the robot">
<status status="NOT RUN" starttime="20220314 11:38:15.535" endtime="20220314 11:38:15.535"/>
</kw>
<kw name="Submit the order">
<status status="NOT RUN" starttime="20220314 11:38:15.536" endtime="20220314 11:38:15.537"/>
</kw>
<kw name="Export Pdf file and take screenshot">
<arg>${row}</arg>
<status status="NOT RUN" starttime="20220314 11:38:15.538" endtime="20220314 11:38:15.538"/>
</kw>
<kw name="Go to order another robot">
<status status="NOT RUN" starttime="20220314 11:38:15.539" endtime="20220314 11:38:15.540"/>
</kw>
<status status="NOT RUN" starttime="20220314 11:38:15.528" endtime="20220314 11:38:15.540"/>
</iter>
<status status="NOT RUN" starttime="20220314 11:38:15.527" endtime="20220314 11:38:15.541"/>
</for>
<kw name="Create a ZIP file of the receipts">
<status status="NOT RUN" starttime="20220314 11:38:15.542" endtime="20220314 11:38:15.542"/>
</kw>
<kw name="Close Browser" library="RPA.Browser.Selenium" type="TEARDOWN">
<doc>Closes the current browser.</doc>
<msg timestamp="20220314 11:38:15.550" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd14b4790&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
<msg timestamp="20220314 11:38:15.552" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd1675f70&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
<msg timestamp="20220314 11:38:15.554" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd14b4400&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
<status status="PASS" starttime="20220314 11:38:15.545" endtime="20220314 11:38:15.560"/>
</kw>
<status status="FAIL" starttime="20220314 11:38:11.257" endtime="20220314 11:38:15.560">Execution terminated by signal</status>
</test>
<test id="s1-t2" name="Auto Pass">
<tag>robot:exit</tag>
<status status="FAIL" starttime="20220314 11:38:15.562" endtime="20220314 11:38:15.567">Test execution stopped due to a fatal error.</status>
</test>
<test id="s1-t3" name="Auto Pass 2">
<tag>robot:exit</tag>
<status status="FAIL" starttime="20220314 11:38:15.581" endtime="20220314 11:38:15.584">Test execution stopped due to a fatal error.</status>
</test>
<test id="s1-t4" name="Auto Pass 3">
<tag>robot:exit</tag>
<status status="FAIL" starttime="20220314 11:38:15.586" endtime="20220314 11:38:15.588">Test execution stopped due to a fatal error.</status>
</test>
<doc>Orders robots from RobotSpareBin Industries Inc.
Saves the order HTML receipt as a PDF file.
Saves the screenshot of the ordered robot.
Embeds the screenshot of the robot to the PDF receipt.
Creates ZIP archive of the receipts and the images.</doc>
<status status="FAIL" starttime="20220314 11:38:06.796" endtime="20220314 11:38:15.590"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="4" skip="0">All Tasks</stat>
</total>
<tag>
<stat pass="0" fail="1" skip="0" info="combined" combined="NOT robot:exit">NOT robot:exit</stat>
</tag>
<suite>
<stat pass="0" fail="4" skip="0" id="s1" name="Tasks">Tasks</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220314 11:38:09.426" level="WARN">This is a deprecated import that will be removed in favor of RPA.Robocorp.Vault</msg>
<msg timestamp="20220314 11:38:15.518" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: Execution terminated by signal</msg>
<msg timestamp="20220314 11:38:15.550" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd14b4790&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
<msg timestamp="20220314 11:38:15.552" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd1675f70&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
<msg timestamp="20220314 11:38:15.554" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x7fccd14b4400&gt;: Failed to establish a new connection: [Errno 111] Connection refused')': /session/401d510fab00adea070f94828d247dd2</msg>
</errors>
</robot>
